pkg/test/matchers/keys.go

```diff
@@ -0,0 +1,107 @@
+package matchers
+
+import (
+	stdErrors "errors"
+	"fmt"
+	"math"
+	"reflect"
+	"runtime/debug"
+	"strings"
+
+	"github.com/onsi/gomega/format"
+	"github.com/onsi/gomega/gstruct/errors"
+	"github.com/onsi/gomega/types"
+)
+
+type allKeysMatcher struct {
+	keyMatcher types.GomegaMatcher
+	failures   []error
+}
+
+func AllKeys(matcher types.GomegaMatcher) types.GomegaMatcher {
```

Comment:
A little comment to explain what this is for would be nice

Reply:
ack

Reply:
addressed